types/testingutils/runner_versioned.go

```diff
@@ -0,0 +1,53 @@
+package testingutils
+
+import (
+	"github.com/bloxapp/ssv-spec/qbft"
+	"github.com/bloxapp/ssv-spec/types"
+)
+
+var SSVDecidingMsgsV = func(consensusData *types.ConsensusData, ks *TestKeySet, role types.BeaconRole) []*types.SSVMessage {
```

Comment:
You are right that the description said all the V suffixed tests should have the "V" removed out of them, since they become the default only option.
However, we noticed that doing so will create lots of changes that will have to be reviewed, slowing us down farther.

So I will change the description.